### DISCUSSION TITLE
storage is not cleared

### DISCUSSION THREAD

**Discussion by kargo123456** [tutao/tutanota#6673] [Bugs]
- [ ] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [ ] I've searched and did not find a similar issue.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - Desktop Environment, if applicable (on Linux)

**Additional context**

**Comment by kargo123456** [comment]
After deleting a large number of emails from the trash, the amount of free storage does not decrease.
I deleted a huge number of emails from the trash, but the storage capacity does not decrease.
I had to move very large letters from other folders to the trash and delete them, but the amount of free storage did not decrease as before
And it's been like this for almost a whole day

**Comment by kargo123456** [comment]
nothing changes
The amount of free storage does not increase after deleting messages

**Comment by rezbyte** [comment]
Hi @kargo123456 ,
Sorry to hear you are having issues with freeing up storage on your account.

If you have a paid plan, please email customer support by using the support dialog inside the Tuta Mail client.

To use the support dialog, press 'Show Help' button (![SupportIcon](https://github.com/tutao/tutanota/assets/48037381/e54cb186-8512-4116-9ef3-7a0d3176f23e)) on the toolbar on the bottom left corner of the screen, then select 'Support'. The support dialog should appear and then you must write your problem into the text box. After some articles appear, scroll to the very bottom of the list & click the button labelled 'Contact support'.

**Comment by kargo123456** [comment]
I have a free plan. The storage is almost completely full, but deleting emails forever does not change anything. The storage is still completely full

**Reply by wilhelmsarahyann-alt** [reply]
I have the same problem. Were you able to solve it?

**Reply by hrb-hub** [reply]
If you're on a free plan and still having this issue, could you please contact `hello[at]tutao.de` so we can look into you specific case?

Cheers :)
